test/images/serve-hostname/serve_hostname.go

```diff
@@ -95,11 +95,11 @@ func main() {
 			log.Fatal(http.ListenAndServe(fmt.Sprintf(":%d", *port), nil))
 		}()
 	}
-
-	// Write to stdout after receiving SIGTERM and SIGINT to help with debugging kubernetes issue #21605
+	log.Printf("Serving on port %d.\n", *port)
 	signals := make(chan os.Signal)
-	signal.Notify(signals, syscall.SIGTERM, syscall.SIGINT)
+	signal.Notify(signals, syscall.SIGTERM)
```

Comment:
can we make this a command line option in case someone needs the old behavior

Reply:
That is just for debugging a flaky test. I do not think anyone still needs it.